source/MRMesh/MRMeshSave.cpp

```diff
@@ -122,19 +124,35 @@ VoidOrErrStr toObj( const Mesh & mesh, const std::filesystem::path & file, const
     if ( !out )
         return unexpected( std::string( "Cannot open file for writing " ) + utf8string( file ) );
 
+#ifndef __EMSCRIPTEN__
```

Comment:
Might be useful to leave a comment that the problem is not the Emscripten platform itself, but the necessity of multiple file handling. It might be rewritten later to be handled in another way (e.g. with a Zip archive).